### ISSUE TITLE
Minor discrepancy between packed 2020a.json and its meta file

### ISSUE THREAD

**Issue by jdatskuid** [moment/moment-timezone#875]
Looks like the packed [2020a.json](url) file was built with an older meta file.

In the [2020a.json meta file](https://github.com/moment/moment-timezone/blob/develop/data/meta/2020a.json#L693), Greenland is associated with the following time zones:

```
"GL": {
	"name": "Greenland",
	"abbr": "GL",
	"zones": [
		"America/Nuuk",
		"America/Danmarkshavn",
		"America/Scoresbysund",
		"America/Thule"
	]
},
```

But in the [2020a.json packed file](https://github.com/moment/moment-timezone/blob/develop/data/packed/2020a.json#L684) we can see that instead of `America/Nuuk`, Greenland is associated with `America/Godthab`:

```
"GL|America/Godthab America/Danmarkshavn America/Scoresbysund America/Thule",
```

I suspect that the 2020a file was built with the [2019c meta file](https://github.com/moment/moment-timezone/blob/develop/data/meta/2019c.json#L689), as Greenland was not yet associated with `America/Nuuk` in that file:

```
"GL": {
	"name": "Greenland",
	"abbr": "GL",
	"zones": [
		"America/Godthab",
		"America/Danmarkshavn",
		"America/Scoresbysund",
		"America/Thule"
	]
},
```

Note that this does not appear to be a problem in the [latest.json packed file](https://github.com/moment/moment-timezone/blob/develop/data/packed/latest.json#L684) (which is versioned as "2020a"):

```
"GL|America/Nuuk America/Danmarkshavn America/Scoresbysund America/Thule",
```

**Comment by gilmoreorless** [comment]
Related: #836

**Comment by ichernev** [comment]
I believe I fixed that in 0.5.36, the data pipeline was messed up, esp around metadata, because the metadata was used but not yet generated, so whatever was latest.js that got picked up (pretty messed up).
